Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-# ruby '2.2.3'
+# ruby '2.3.3'
```

Comment:
Should be able to uncomment this now.

Reply:
I believe this was a memory aid to tell which version of ruby ... I'm leaving it

Reply:
I don't understand. In the original version of the code, this line is commented out. What is the effect of uncommenting it?

Reply:
You can specify a ruby version in your Gemfile, and it will raise an error if the app is loaded with a different version.

https://devcenter.heroku.com/articles/ruby-versions

Reply:
@Wilfongjt This is something I temporarily commented out my in experimental branch.  It wasn't commented out in the original version of the code.  The purpose of my experimental branch was to just get it running asap on Heroku so the LGROW team could review it.  I hadn't intended those changes to be merged in like they were.    That's where the confusion originated from.   You can review my experimental PR here: https://github.com/citizenlabsgr/adopt-a-drain/pull/13 (in case you want to see what else got introduced)